pkg/client/interfaces.go

```diff
@@ -67,10 +67,6 @@ type DeleteConfig struct {
 
 // RetrieveConfig are the options passed to RetrieveResults.
 type RetrieveConfig struct {
-	// CmdErr is the place to write errors to.
-	CmdErr io.Writer
-	// Errc reports errors from go routines that retrieve may spawn.
-	Errc chan error
 	// Namespace is the namespace the sonobuoy aggregator is running in.
 	Namespace string
```

Comment:
in other cases where we only pass in a namespace (`getStatus`) we don't wrap it in a struct. We should change this case or the other one for consistency